.circleci/export-playwright-env.sh

```diff
@@ -3,68 +3,68 @@
 
 #DSM
 export dsmUser=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.users | .[] | select(.app==\"dsm\") | .userName")
 export dsmUserPassword=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.users | .[] | select(.app==\"dsm\") | .password")
-echo "export DSM_USER_EMAIL=$dsmUser" >>playwright-env/envvars
-echo "export DSM_USER_PASSWORD=$dsmUserPassword" >>playwright-env/envvars
+echo "export DSM_USER_EMAIL=$dsmUser" >> playwright-env/envvars
+echo "export DSM_USER_PASSWORD=$dsmUserPassword" >> playwright-env/envvars
 
 # SINGULAR
 export singularUser=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.users | .[] | select(.app==\"singular\") | .userName")
 export singularUserPassword=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.users | .[] | select(.app==\"singular\") | .password")
-echo "export SINGULAR_USER_EMAIL=$singularUser" >>playwright-env/envvars
-echo "export SINGULAR_USER_PASSWORD=$singularUserPassword" >>playwright-env/envvars
+echo "export SINGULAR_USER_EMAIL=$singularUser" >> playwright-env/envvars
+echo "export SINGULAR_USER_PASSWORD=$singularUserPassword" >> playwright-env/envvars
 
 # RGP
 export rgpUser=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.users | .[] | select(.app==\"rgp\") | .userName")
 export rgpUserPassword=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.users | .[] | select(.app==\"rgp\") | .password")
-echo "export RGP_USER_PASSWORD=$rgpUserPassword" >>playwright-env/envvars
-echo "export RGP_USER_EMAIL=$rgpUser" >>playwright-env/envvars
+echo "export RGP_USER_PASSWORD=$rgpUserPassword" >> playwright-env/envvars
+echo "export RGP_USER_EMAIL=$rgpUser" >> playwright-env/envvars
 
 # Read Auth0 RGP client credentials
-export rgpDomain=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .domain")
-export rgpAudience=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .audience")
-export rgpClientId=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .clientId")
-export rgpClientSecret=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .clientSecret")
-echo "export RGP_AUTH0_DOMAIN=$rgpDomain" >>playwright-env/envvars
-echo "export RGP_AUTH0_AUDIENCE=$rgpAudience" >>playwright-env/envvars
-echo "export RGP_AUTH0_CLIENT_ID=$rgpClientId" >>playwright-env/envvars
-echo "export RGP_AUTH0_CLIENT_SECRET=$rgpClientSecret" >>playwright-env/envvars
+export rgpDomain=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"$ENV\") | .domain")
```

Comment:
Replace `<< parameters.env >>` with `$ENV`.

Reply:
I think that makes it more read-able - looks nice